tensorflow/python/util/compat.py

```diff
@@ -118,12 +122,14 @@ def path_to_str(path):
   """Returns the file system path representation of a `PathLike` object, else as it is.
 
   Args:
-    path: An object that can be converted to path representation.
+    path: One or more objects that can be converted to path representation.
 
   Returns:
-    A `str` object.
+    One or more `str` object.
```

Comment:
Can you modify the `Args:` and `Returns:` sections of this docstring to mention the behavior on non-pathlike objects?